config.yaml

```diff
@@ -738,3 +738,4 @@ individual_repositories:
   - https://github.com/ytx1991/ChiaSimplePlotter
   - https://github.com/999cross999/hiveos-hpool-chia
   - https://github.com/farmerdiamonds/splash-nodejs
+  - https://github.com/warpdotgreen
```

Comment:
This takes individual repository names, not organizations. Mostly out of caution because not every repository someone has may be related to Chia.